src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/AStar.java

```diff
@@ -0,0 +1,100 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.Map;
+
+public class AStar {
+    private static final NodeBase nodeBase = NodeBase.getInstance();
+
+  public static double getDistanceFromCurrentNodeToEndOrStartNode(NodeArm point, NodeArm current){
```

Comment:
what is the difference between getDistanceFromCurrentNodeToEndOrStartNode and getDistanceBetweenToPoints

Reply:
Changed it, in the func who call this func i used getDistanceBetweenTwoPoints instead